tests/tests/fv3_ccpp_gfs_v16beta_RRTMGP_debug

```diff
@@ -1,10 +1,10 @@
 ###############################################################################
 #
-#  FV3 CCPP GFS v16beta w/ RRTMGP compiled with 32-bit dynamics test in DEBUG mode
+#  FV3 CCPP GFS v16beta w/ RRTMGP test in DEBUG mode
 #
 ###############################################################################
 
-export TEST_DESCR="Run FV3 32bit CCPP GFS v16beta w/ RRTMGP in DEBUG mode"
+export TEST_DESCR="Run FV3 CCPP GFS v16beta w/ RRTMGP in DEBUG mode"
 
 export CNTL_DIR=fv3_gfs_v16beta_debug_RRTMGP
```

Comment:
Last ones: change `export CNTL_DIR=fv3_gfs_v16beta_debug_RRTMGP` to `export CNTL_DIR=fv3_gfs_v16beta_RRTMGP_debug` (and similar for GFS v15p2 debug) so that the run directories are consistent with the test names, please - avoids confusion.